tests/test_visitors/test_ast/test_naming/test_naming_rules/test_wrong_names.py

```diff
@@ -1,11 +1,41 @@
 import pytest
 
 from wemake_python_styleguide.violations.naming import (
+    UpperCaseAttributeViolation,
     WrongVariableNameViolation,
 )
 from wemake_python_styleguide.visitors.ast.naming import WrongNameVisitor
 
 
+@pytest.mark.parametrize('wrong_name', [
+    'SOME',
```

Comment:
It looks like we are missing a template in our cases here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_naming/conftest.py

```python
class_name = 'class {0}: ...'
```

Can you please also add it?

Reply:
sure => a90b5ae